internal/observability/event/context.go

```diff
@@ -136,7 +142,10 @@ func WriteError(ctx context.Context, caller Op, e error, opt ...Option) {
 	if !ok {
 		eventer = SysEventer()
 		if eventer == nil {
-			if !ci {
+			switch {
+			case !ci:
+			case !isTest:
```

Comment:
Nit: what do you think about
```suggestion
			case !ci, !isTest:
```

Reply:
Of course, I did this intentionally as well since I find it easier to read.